internal/postprocessor/main.go

```diff
@@ -193,15 +197,72 @@ func (c *config) RegenSnippets() error {
 	return nil
 }
 
-func (c *config) replaceAllForSnippets(googleCloudDir, snippetDir string) error {
-	return execv.ForEachMod(googleCloudDir, func(dir string) error {
+func (c *config) parseAPIShortnames(confs []*generator.MicrogenConfig, manualEntries []generator.ManifestEntry) (map[string]string, error) {
+	shortnames := map[string]string{}
+	runConfs := confs
```

Comment:
If you need to copy a slice you almost always want to copy the elments explicitly or you can have unintended behavior creep in: https://go.dev/play/p/gkUJknLHYiU